resources/grafana/downstream/grafana-dashboard-dataplane-cluster-view.yaml

```diff
@@ -2940,13 +2943,13 @@ data:
         ]
       },
       "time": {
-        "from": "now-24h",
+        "from": "now-6h",
```

Comment:
How is 6 hr time frame can be helpful to debug an issue?
The idea of having this dashboard is to observe the overall status and 24 hr of time acts as a standard. I think that the specific user can anytime change the time frame to debug an issue or for any reason. 
Did you get this requirement specifically from the SRE team?

Reply:
reverted https://github.com/bf2fc6cc711aee1a0c2a/cos-observability-resources/pull/23/commits/8636023eabbbf6063efda80ff9cecde3cd177dc3